ush/parsing_namelists_FV3.sh

```diff
@@ -0,0 +1,619 @@
+/
+
+&external_ic_nml
+  filtered_terrain = $filtered_terrain
+  levp = $LEVS
+  gfs_dwinds = $gfs_dwinds
+  checker_tr = .false.
+  nt_checker = 0
+  $external_ic_nml
+/
+
+&gfs_physics_nml
+  fhzero       = $FHZER
+  h2o_phys     = ${h2o_phys:-".true."}	
+  ldiag3d      = ${ldiag3d:-".false."}
+  fhcyc        = $FHCYC			
+  use_ufo      = ${use_ufo:-".true."}
+  pre_rad      = ${pre_rad:-".false."}
+  ncld         = ${ncld:-1}		
+  imp_physics  = ${imp_physics:-"99"}	
+EOF
+
+case "${CCPP_SUITE:-}" in
+  "FV3_GFS_v15p2_coupled")
+  cat >> input.nml << EOF
+  oz_phys      = .false.
+  oz_phys_2015 = .true.
+EOF
+  ;;
+  "FV3_GSD_v0")
+  cat >> input.nml << EOF
+  iovr         = ${iovr:-"3"}
+  ltaerosol    = ${ltaerosol:-".F."}    
+  lradar       = ${lradar:-".F."}	
+  ttendlim     = ${ttendlim:-0.005}
+  oz_phys      = ${oz_phys:-".false."}
+  oz_phys_2015 = ${oz_phys_2015:-".true."}
+  lsoil_lsm    = ${lsoil_lsm:-"4"}
+  do_mynnedmf  = ${do_mynnedmf:-".false."}
+  do_mynnsfclay = ${do_mynnsfclay:-".false."}
+  icloud_bl    = ${icloud_bl:-"1"}	! In config.fcst
+  bl_mynn_edmf = ${bl_mynn_edmf:-"1"}	! In config.fcst
+  bl_mynn_tkeadvect=${bl_mynn_tkeadvect:-".true."}	! In config.fcst
+  bl_mynn_edmf_mom=${bl_mynn_edmf_mom:-"1"}	! In config.fcst
+  min_lakeice  = ${min_lakeice:-"0.15"}
+  min_seaice   = ${min_seaice:-"0.15"}
+EOF
+  ;;
+  FV3_GFS_v16_coupled*)
+  cat >> input.nml << EOF
+  iovr         = ${iovr:-"3"}
+  ltaerosol    = ${ltaerosol:-".false."}
+  lradar       = ${lradar:-".false."}
+  ttendlim     = ${ttendlim:-"0.005"}
+  oz_phys      = ${oz_phys:-".false."}
+  oz_phys_2015 = ${oz_phys_2015:-".true."}
+  do_mynnedmf  = ${do_mynnedmf:-".false."}
+  do_mynnsfclay = ${do_mynnsfclay:-".false."}
+  icloud_bl    = ${icloud_bl:-"1"}
+  bl_mynn_edmf = ${bl_mynn_edmf:-"1"}
+  bl_mynn_tkeadvect = ${bl_mynn_tkeadvect:-".true."}
+  bl_mynn_edmf_mom = ${bl_mynn_edmf_mom:-"1"}
+  min_lakeice  = ${min_lakeice:-"0.15"}
+  min_seaice   = ${min_seaice:-"0.15"}
+EOF
+  ;;
+  FV3_GFS_v16*)
+  cat >> input.nml << EOF
+  iovr         = ${iovr:-"3"}
+  ltaerosol    = ${ltaerosol:-".false."}
+  lradar       = ${lradar:-".false."}
+  ttendlim     = ${ttendlim:-"0.005"}
+  oz_phys      = ${oz_phys:-".false."}
+  oz_phys_2015 = ${oz_phys_2015:-".true."}
+  lsoil_lsm    = ${lsoil_lsm:-"4"}
+  do_mynnedmf  = ${do_mynnedmf:-".false."}
+  do_mynnsfclay = ${do_mynnsfclay:-".false."}
+  icloud_bl    = ${icloud_bl:-"1"}
+  bl_mynn_edmf = ${bl_mynn_edmf:-"1"}
+  bl_mynn_tkeadvect = ${bl_mynn_tkeadvect:-".true."}
+  bl_mynn_edmf_mom = ${bl_mynn_edmf_mom:-"1"}
+  min_lakeice  = ${min_lakeice:-"0.15"}
+  min_seaice   = ${min_seaice:-"0.15"}
+EOF
+  ;;
+  *)
+  cat >> input.nml << EOF
+  iovr         = ${iovr:-"3"}
+EOF
+  ;;
+esac
+
+cat >> input.nml <<EOF
+  pdfcld       = ${pdfcld:-".false."}
+  fhswr        = ${FHSWR:-"3600."}
+  fhlwr        = ${FHLWR:-"3600."}
+  ialb         = ${IALB:-"1"}           ! In config.fcst
```

Comment:
Are the "! In config.fcst" crow related or also valid comments w/out crow?

Reply:
This is also cleaned up now.